string-plugin/src/test/java/com/github/underscore/string/StringTest.java

```diff
@@ -1986,6 +1986,8 @@ public void chain() {
             public Boolean apply(String str) { return true; } });
         $.chain(new String[] {""}).rejectIndexed(new PredicateIndexed<String>() {
             public boolean apply(int index, String str) { return true; } });
+        $.chain(new String[] {""}).filterFalse(new Predicate<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.filterFalse\(Predicate\) ignored, is this OK in com\.github\.underscore\.string\.StringTest\.chain\(\)